packages/vue-instantsearch/src/components/__tests__/Hits.js

```diff
@@ -28,6 +42,37 @@ it('accepts an escapeHTML prop', () => {
   expect(wrapper.vm.widgetParams.escapeHTML).toBe(true);
 });
 
+it('exposes banner prop to the banner slot', () => {
+  __setState({
+    ...defaultState,
+    banner: {
+      image: {
+        urls: [{ url: 'https://via.placeholder.com/550x250' }],
+      },
+    },
+  });
+
+  const wrapper = mount(
+    {
+      components: { Hits },
+      template: `
+      <Hits>
+        <template v-slot:banner="{ banner }">
+          <img :src=banner.image.urls[0].url />
+        </template>
+      </Hits>
+    `,
+    },
+    {
+      propsData: {
+        showBanner: true,
+      },
+    }
+  );
+  const img = wrapper.find('img');
+  expect(img.attributes('src')).toBe('https://via.placeholder.com/550x250');
+});
+
```

Comment:
This test runs locally, but [fails in CI](https://app.circleci.com/pipelines/github/algolia/instantsearch/11204/workflows/fa6f843a-c2a3-4823-8645-738773317cd0/jobs/54863) - so I'm assuming there is some difference between vue2 and vue3 that isn't accounted for. I added it as almost an afterthought. Since the common tests succeed we could remove this unless there is a straight-forward approach to ensuring it runs for vue3 as well.

Reply:
That's because the thing with `Fragment` from `preact` doesn't work in Vue 3, implementing what I describe in my comment should make it work

Reply:
Makes sense. Worked as expected now that `Fragment` has been removed.